### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by BlueBeret** [JustDoom/Better-Messages#3]
I found this plugin and it seems have a good review. I just installed it, but I don't know how to set up this plugin (like editing message, what kind of message I can edit). I looked everywhere and what I found is just this `/bm`.  

Where I can find all available command or editable message?

**Comment by BlueBeret** [comment]
ok, found it, it's in plugins folder lmao. mb :D

**Comment by JustDoom** [comment]
Yeah I need to work on some proper documentation but for now everything you need should be explained in the config file. I'll keep this open until then

**Comment by hikinata** [comment]
how install it on Bungeecord ?

**Comment by JustDoom** [comment]
How you install anything on it. Drag the plugin over. Make sure it is the bungeecord file. Then install the normal plugin on each server you want it working on

**Comment by hikinata** [comment]
Hmm i make it but i have an error 

"Error loading plugin BetterMessages
java.lang.NoClassDefFoundError: org/bukkit/plugin/java/JavaPlugin"

**Comment by JustDoom** [comment]
It seems like you did not download the bungeecord version. Also please either join my discord or open a new GitHub issue
Make sure to install the BUNGEECORD one on Bungeecord and the SPIGOT one on Spigot/Paper or whatever other fork you use. Also please provide more information on errors next time
https://modrinth.com/plugin/bettermessages/version/3.3.2

**Comment by hikinata** [comment]
right , i have use "select game version" .. perfect ! but I must configure only config on bungee ?

**Comment by JustDoom** [comment]
The config is per server, the bungeecord plugin only bridges servers together. If you need more help please open a new issue
